### DISCUSSION TITLE
Remove "Login with email" from Dex WebUI

### DISCUSSION THREAD

**Discussion by vovka17surnyk** [dexidp/dex#3357] [Q&A]
Hi guys, 

is there is way to Remove "Login with email" from Dex WebUI, as I`m going to use only GitLab?

**Comment by nabokihms** [comment]
Yes! Set `enablePasswordDB: false` in your Dex config.
https://github.com/dexidp/dex/blob/80d530d9bf0e38dfb0549c847c92e3c697b0402b/examples/config-dev.yaml#L146
